### PR TITLE
Various cleanup

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
##### Description of change
- discard one unused variable, 
- remove one useless,
- disable some old dead Err defines, and discard associated macros.

Targets:
- [X] master
- [X] branch 1.1.0
- [X] branch 1.0.2 ( p5_pbev2.c only concerned, and if you want)

### CODE REVIEW

File: crypto/asn1/p5_pbev2.c [hunk 1]
```diff
@@ -41,27 +41,25 @@ X509_ALGOR *PKCS5_pbe2_set_iv(const EVP_CIPHER *cipher, int iter,
                               unsigned char *salt, int saltlen,
                               unsigned char *aiv, int prf_nid)
 {
-    X509_ALGOR *scheme = NULL, *kalg = NULL, *ret = NULL;
+    X509_ALGOR *scheme, *ret = NULL;
```
Comment (levitte) [inline crypto/asn1/p5_pbev2.c]: Please keep the initialization of `scheme`

Comment (FdaSilvaYY) [inline crypto/asn1/p5_pbev2.c]: Fixed !

File: include/openssl/err.h [hunk 2]
```diff
@@ -84,12 +84,12 @@ typedef struct err_state_st {
 # define ERR_LIB_COMP            41
 # define ERR_LIB_ECDSA           42
 # define ERR_LIB_ECDH            43
-# define ERR_LIB_STORE           44
```
Comment (richsalz) [inline include/openssl/err.h]: don't edit this file; these things will come back when someone does 'make errs'

Comment (FdaSilvaYY) [inline include/openssl/err.h]: Not sure, but who do you explain that some defines (like 15-lines or more above) are commented ?
I have not yet  dig the pelr/mkerr code.

Comment (richsalz) [inline include/openssl/err.h]: oh right, i was confused and thinking of error codes.

**Comment by richsalz** [comment]
+1

**Comment by FdaSilvaYY** [comment]
@levitte : changes done .
PR rebased.

**Comment by levitte** [comment]
Merged (I did the port to 1.0.2 as well)

**Comment by FdaSilvaYY** [comment]
Thanks !

**Pull request closed without merging**